### PR TITLE
Add Greekleaks, move HuffPost to exemptions

### LINKED ISSUE #184: Add Short Onion Name for Greekleaks
Please create the short onion name `greekleaks.securedrop.tor.onion` for Greekleaks

Directory entry: https://securedrop.org/directory/greekleaks/
Long onion address: jatasaqcoe7lqdpcyxo7vl3e5tdvl5jgmtadfat77i25qdj6z6a4ulad.onion

Thank you!

### PR DESCRIPTION
**Pull request by legoktm**
Fixes #184.

### Status

Ready for review

### Review Checklist

- [x] Changes to `onboarded.txt` are accurate
- [x] The file `default.rulesets.TIMESTAMP.gz` has been updated, extracting that file and inspecting the contents of the JSON file produces the expected rules
- [x] The ruleset has been verified by modifying the HTTPS Everywhere configuration in a Tor Browser instance pointing to `Path Prefix`: `https://raw.githubusercontent.com/freedomofpress/securedrop-https-everywhere-ruleset/$BRANCH_NAME`
- [x] `index.html` has been updated using `./update_index.sh`

### Post-Deployment Checklist

- [ ] Added/modified onion names have been updated in the SecureDrop Directory

### CODE REVIEW

**Comment by nathandyer** [comment]
Not sure why, but so far `greekleaks.securedrop.tor.onion` is not working for me after updating the ruleset in Tor Browser (via `about:rulesets`) to: `https://raw.githubusercontent.com/freedomofpress/securedrop-https-everywhere-ruleset/greekleaks` 

I verified it's seeing the ruleset, and that it's updating them (it checks and updates immediately after I enter it), and I am able to open other SecureDrop source interfaces via their Short Onion Names.

Anything I'm missing? Or is something perhaps broken here?

File: rulesets/greekleaks-securedrop-ruleset.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<ruleset name="Greekleaks">
+	<target host="greekleaks.securedrop.tor.onion" />
+	<rule from="^http[s]?://greekleaks.securedrop.tor.onion"
+    to="http://http://jatasaqcoe7lqdpcyxo7vl3e5tdvl5jgmtadfat77i25qdj6z6a4ulad.onion" />
```
Comment (legoktm) [inline rulesets/greekleaks-securedrop-ruleset.xml:4]: Uhoh, this is a double http somehow

Comment (nathandyer) [reply rulesets/greekleaks-securedrop-ruleset.xml:4]: @legoktm Of course - done!

**Review by nathandyer — APPROVED** [review]
Thanks for that quick fix @legoktm! Happy to report it's working now, and everything looks correct.

**Pull request merged**